pom.xml

```diff
@@ -1800,6 +1800,12 @@ under the License.
         <version>3.2.0</version>
```

Comment:
It would be great if you could define a property like as
```xml
<mockito.version>3.2.4</mockito.version>
```
right below line 454, e.g. after
```xml
<junit.version>5.5.2</junit.version>
```

and then reference that both for `mockito-core` and `mockito-junit-jupiter`'s versions.

Reply:
Sure, done that.